### PR TITLE
feat(appeals): lpa questionnaire submission command

### PR DESCRIPTION
**Pull request by Si3rr4wow**

### CODE REVIEW

File: src/schemas.d.ts [hunk 1]
```diff
@@ -1647,6 +1647,161 @@ export interface Address {
...
+           */
+          neighbouringSiteAddressCounty: string | null;
+          /**
+           * Postal code of the site address
+           */
+          neighbouringSiteAddressPostcode: string;
+          /**
+           * Provided information on site accessibility on this address
+           */
+          neighbouringSiteAccessDetails: string | null;
```
Comment (bryn-pins) [inline src/schemas.d.ts:1727]: don't think we currently ask about neighbour site access/safety details

File: src/schemas.d.ts [hunk 2]
```diff
@@ -1647,6 +1647,161 @@ export interface Address {
...
+        }[]
+      | null;
+    /**
+     * A list of affected listed building IDs from Historic England
+     */
+    affectedListedBuildingNumbers: string[] | null;
+    /**
+     * Indicates if the LPA has applied for costs
+     */
+    lpaCostsAppliedFor: boolean | null;
```
Comment (bryn-pins) [inline src/schemas.d.ts:1742]: don't think we ask this to LPAs

File: src/schemas.d.ts [hunk 3]
```diff
@@ -1647,6 +1647,161 @@ export interface Address {
...
+     */
+    caseReference: string;
+    /**
+     * The date the LPA provided a response to the case
+     */
+    lpaQuestionnaireSubmittedDate: string | null;
+    /**
+     * A statement provided by the LPA
+     */
+    lpaStatement: string | null;
```
Comment (bryn-pins) [inline src/schemas.d.ts:1666]: not sure we ask for an lpa statement, unless I'm we've called this something else

**Review by bryn-pins — COMMENTED** [review]
some thoughts on the spec

**Review by bryn-pins — DISMISSED** [review]
I'm happy

**Review by AlexMoulin — APPROVED** [review]

**Pull request merged**